npx missing 'fs-extra' module

# Expected behavior
Commands via npx run without error. 

# Current behavior
When running `npx office-addin-manifest validate manifest.xml`, npx installs `office-addin-manifest` then throws an error "Cannot find module 'fs-extra'".

## Context
It seems likely to me that since `fs-extra` is [listed as a devDependency in `package.json`](https://github.com/OfficeDev/Office-Addin-Scripts/blob/be2d3616744f60bd05f9813144993d4b39dae29e/packages/office-addin-manifest/package.json#L44) it does not get installed by npx. After running `npm i fs-extra` in `/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/`, the validate command works.

* Operating System: MacOS 12.0.1
* Node version: v16.13.0
* Office version: 
* Tool version: 1.7.8

## Failure Logs

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'fs-extra'
Require stack:
- /Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/export.js
- /Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/commands.js
- /Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/cli.js
- /Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at Module.patchedRequire [as require] (/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/diagnostic-channel/dist/src/patchRequire.js:14:46)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/export.js:13:17)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/export.js',
    '/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/commands.js',
    '/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/lib/cli.js',
    '/Users/mark/.npm/_npx/4692618cb69c9be3/node_modules/office-addin-manifest/cli.js'
  ]
}
```

--- a/packages/office-addin-manifest/package.json
+++ b/packages/office-addin-manifest/package.json
@@ -29,7 +29,8 @@
     "office-addin-usage-data": "^1.4.7",
     "path": "^0.12.7",
     "uuid": "^8.3.2",
-    "xml2js": "^0.4.23"
+    "xml2js": "^0.4.23",
+    "fs-extra": "^7.0.1"
   },
   "devDependencies": {
     "@types/fs-extra": "^9.0.13",
@@ -41,7 +42,6 @@
     "@types/xml2js": "^0.4.5",
     "concurrently": "^6.2.2",
     "copy-dir": "^0.4.0",
-    "fs-extra": "^7.0.1",
     "mocha": "^9.1.1",
     "office-addin-lint": "^2.0.1",
     "rimraf": "^3.0.2",

--- a/packages/office-addin-manifest/package.json
+++ b/packages/office-addin-manifest/package.json
@@ -24,13 +24,13 @@
   "dependencies": {
     "chalk": "^2.4.2",
     "commander": "^6.2.0",
+    "fs-extra": "^7.0.1",
     "jszip": "^3.7.1",
     "node-fetch": "^2.6.1",
     "office-addin-usage-data": "^1.4.7",
     "path": "^0.12.7",
     "uuid": "^8.3.2",
-    "xml2js": "^0.4.23",
-    "fs-extra": "^7.0.1"
+    "xml2js": "^0.4.23"
   },
   "devDependencies": {
     "@types/fs-extra": "^9.0.13",

